frontend/www/js/omegaup/components/common/Navbar.vue

```diff
@@ -212,6 +212,9 @@
                 <li>
                   <a href="/badge/list/">{{ T.navViewBadges }}</a>
                 </li>
+                <li>
+                  <a href="/course/mine">{{ T.myCourses }} </a>
```

Comment:
Mejor hay que cambiarle el nombre a este string
```suggestion
                  <a href="/course/mine">{{ T.navMyCourses }} </a>
```